Library/Formula/pacapt.rb

```diff
@@ -0,0 +1,19 @@
+# Pacapt is a wrapper for all major package managers. It allows
+# a user to use only one pacman-likes syntax while working with
+# different package managers.
+
+class Pacapt < Formula
+
+  homepage "https://github.com/letalvoj/pacapt" 
+  url "https://github.com/icy/pacapt.git", :using => :git, :tag => "v2.0.2"
```

Comment:
Also needs a `:revision` but using the tag archive would be better unless there are submodules. Also the `:using` isn't necessaryl